project/StreamOperatorsIndexGenerator.scala

```diff
@@ -111,7 +112,8 @@ object StreamOperatorsIndexGenerator extends AutoPlugin {
       "fromGraph",
       "actorSubscriber",
       "foldAsync",
-      "newOnCompleteStage"))
+      "newOnCompleteStage",
+      "exists"))
```

Comment:
I think this is not needed, as https://github.com/apache/incubator-pekko/pull/989/